.matrix.yml

```diff
@@ -3,6 +3,44 @@ JENKINS_TESTTYPES:
   - univention-bareos
 
 OS:
+  Sanitize:
+    "F37":
+      TYPE: scripted
+      IMAGE: fedora37
+      BUILD_SCRIPT: CD/sanitize/build-sanitize.sh
+      FINISH_SCRIPT: CD/sanitize/finish-sanitize.sh
+      BUILD_FOR_VERSIONS:
+        - bareos-22
```

Comment:
```suggestion
```
This is not used anymore if the matrix.yml is in the Bareos repository.

Reply:
Done